pkg/connector/login.go

```diff
@@ -0,0 +1,188 @@
+			ID:          "cookies",
+		},
+	}
+}
+
+func (tc *TwitterConnector) CreateLogin(_ context.Context, user *bridgev2.User, flowID string) (bridgev2.LoginProcess, error) {
+	if flowID != "cookies" {
+		return nil, fmt.Errorf("unknown login flow ID: %s", flowID)
+	}
+	return &TwitterLogin{User: user}, nil
+}
+
+func (t *TwitterLogin) Start(_ context.Context) (*bridgev2.LoginStep, error) {
+	return &bridgev2.LoginStep{
+		Type:         bridgev2.LoginStepTypeCookies,
+		StepID:       "fi.mau.twitter.login.enter_cookies",
+		Instructions: "Enter a JSON object with your cookies, or a cURL command copied from browser devtools.\n\nFor example: `{\"ct0\":\"123466-...\",\"auth_token\":\"abcde-...\"}`",
+		CookiesParams: &bridgev2.LoginCookiesParams{
+			URL:       "https://x.com",
+			UserAgent: "",
+			Fields: []bridgev2.LoginCookieField{
+				{
+					ID:       "ct0",
+					Required: true,
+					Sources: []bridgev2.LoginCookieFieldSource{
+						{Type: bridgev2.LoginCookieTypeCookie, Name: "ct0"},
+					},
+				},
+				{
+					ID:       "auth_token",
+					Required: true,
+					Sources: []bridgev2.LoginCookieFieldSource{
+						{Type: bridgev2.LoginCookieTypeCookie, Name: "auth_token"},
+					},
+				},
+				/*
+						{
+							ID: "guest_id",
+							Required: false,
+							Sources: []bridgev2.LoginCookieFieldSource{
+								{ Type: bridgev2.LoginCookieTypeCookie, Name: "guest_id" },
+							},
+						},
+						{
+							ID: "twid",
+							Required: false,
+							Sources: []bridgev2.LoginCookieFieldSource{
+								{ Type: bridgev2.LoginCookieTypeCookie, Name: "twid" },
+							},
+						},
+						{
+							ID: "kdt",
+							Required: false,
+							Sources: []bridgev2.LoginCookieFieldSource{
+								{ Type: bridgev2.LoginCookieTypeCookie, Name: "kdt" },
+							},
+						},
+						{
+							ID: "night_mode",
+							Required: false,
+							Sources: []bridgev2.LoginCookieFieldSource{
+								{ Type: bridgev2.LoginCookieTypeCookie, Name: "night_mode" },
+							},
+						},
+						{
+							ID: "personalization_id",
+							Required: false,
+							Sources: []bridgev2.LoginCookieFieldSource{
+								{ Type: bridgev2.LoginCookieTypeCookie, Name: "personalization_id" },
+							},
+						},
+						{
+							ID: "guest_id_marketing",
+							Required: false,
+							Sources: []bridgev2.LoginCookieFieldSource{
+								{ Type: bridgev2.LoginCookieTypeCookie, Name: "guest_id_marketing" },
+							},
+						},
+						{
+							ID: "guest_id_ads",
+							Required: false,
+							Sources: []bridgev2.LoginCookieFieldSource{
+								{ Type: bridgev2.LoginCookieTypeCookie, Name: "guest_id_ads" },
+							},
+						},
+						{
+							ID: "d_prefs",
+							Required: false,
+							Sources: []bridgev2.LoginCookieFieldSource{
+								{ Type: bridgev2.LoginCookieTypeCookie, Name: "d_prefs" },
+							},
+						},
+					},
+				*/
+			},
+		},
+	}, nil
+}
+
+func (t *TwitterLogin) Cancel() {
+	//TODO implement me
+	panic("implement me")
+}
```

Comment:
Looks like there are no persistent connections, so cancel can just be a no-op

```suggestion
func (t *TwitterLogin) Cancel() {}
```